core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -104,6 +104,9 @@ private[spark] object Utils extends Logging {
   val MAX_DIR_CREATION_ATTEMPTS: Int = 10
   @volatile private var localRootDirs: Array[String] = null
 
+  /** Root directory for Apache Spark documentation. */
+  val DOC_ROOT_DIR = "https://spark.apache.org/docs/latest"
```

Comment:
Have you considered other places for the value? For instance, shouldn't it be placed in SparkBuildInfo:
https://github.com/apache/spark/blob/0494dc90af48ce7da0625485a4dc6917a244d580/core/src/main/scala/org/apache/spark/package.scala#L98

@HyukjinKwon @dongjoon-hyun @srowen @cloud-fan WDYT?

Reply:
+1

Reply:
+1, too

Reply:
+1

Reply:
Updated. Thanks!! :-)